### PR TITLE
Review

### PR DESCRIPTION
**Pull request by sayakpaul**

### CODE REVIEW

File: deploy-vertex-ai.md [hunk 1]
```diff
@@ -0,0 +1,525 @@
+---
+title: Deploying 🤗 ViT on Vertex AI
+thumbnail: /blog/assets/97_vertex_ai/thumb.png
```
Comment (sayakpaul) [inline deploy-vertex-ai.md:3]: To be updated.

File: deploy-vertex-ai.md [hunk 2]
```diff
@@ -0,0 +1,525 @@
...
+The [`google-cloud-aiplatform`](https://pypi.org/project/google-cloud-aiplatform/)
+Python SDK provides easy APIs to manage the lifecycle of a deployment on
+Vertex AI. It is divided into four steps - **1)** uploading a model,
+**2)** creating an endpoint, **3)** deploying the model to the endpoint,
+and **4)** making prediction requests. Throughout these steps, you will
+need `ModelServiceClient`, `EndpointServiceClient`, and
+`PredictionServiceClient` modules from the `google-cloud-aiplatform`
+Python SDK to interact with Vertex AI.
+
+![](./assets/97_vertex_ai/image3.png)
```
Comment (sayakpaul) [inline deploy-vertex-ai.md:193]: @deep-diver this image didn't render correctly. Could you take a look?

File: deploy-vertex-ai.md [hunk 3]
```diff
@@ -0,0 +1,525 @@
...
+```
+
+Let’s unpack the code piece by piece:
+
+- `GCS_BUCKET` denotes the path of your GCS bucket where the model
+  artifacts are located (e.g., `gs://hf-tf-vision`).
+
+- In `container_spec`, you provide the URI of a Docker image that will
+  be used to serve predictions. Vertex AI
+  [<u>provides</u>](https://cloud.google.com/vertex-ai/docs/predictions/pre-built-containers)
```
Comment (deep-diver) [inline deploy-vertex-ai.md:220]: maybe the link should be on `pre-built images`?

File: deploy-vertex-ai.md [hunk 4]
```diff
@@ -0,0 +1,525 @@
...
+
+- `machine_spec` lets you define the configurations of the deployment
+  hardware:
+
+  - `machine_type` is the base machine type that will be used to run
+    the Docker image. The underlying autoscaler will scale this
+    machine as per the traffic load.
+
+  - `accelerator_type` is the hardware accelerator that will be used
+    > to perform inference.
```
Comment (deep-diver) [inline deploy-vertex-ai.md:295]: `>` is a typo?

File: deploy-vertex-ai.md [hunk 5]
```diff
@@ -0,0 +1,525 @@
...
+
+  - `accelerator_count` denotes the number of hardware accelerators to
+    > attach to each replica.
+
+**Note** that providing an accelerator is not a requirement to deploy
```
Comment (deep-diver) [inline deploy-vertex-ai.md:300]: `>` is a typo?

File: deploy-vertex-ai.md [hunk 6]
```diff
@@ -0,0 +1,525 @@
...
+It takes about 15 - 30 minutes for Vertex AI to scope the deployment.
+Once it’s done, you should be able to see it on the
+[<u>console</u>](https://console.cloud.google.com/vertex-ai/endpoints).
+
+# Performing Predictions
+
+Assuming you succeeded in the above step, you can test the deployed
+Endpoint by making a prediction request.
+
+First, prepare a base64 encoded image string:
```
Comment (deep-diver) [inline deploy-vertex-ai.md:330]: maybe leave a link what base64 is a good idea
: https://en.wikipedia.org/wiki/Base64

Comment (sayakpaul) [reply deploy-vertex-ai.md:330]: has been provided before. Refer to the text `The model will accept ...`.

**Comment by deep-diver** [comment]
Looks good!

**Comment by sayakpaul** [comment]
@deep-diver could you also update the thumbnail?

**Comment by deep-diver** [comment]
@sayakpaul image1 is meant to be the thumbnail, no?

**Pull request merged**